README.md

```diff
@@ -1,10 +1,19 @@
-# mumble-pahelper
+<p align="center"><img src = "resources/MumblePAHelper.png"/></p>
 
-Mumble Positional Audio Helper is a small tool for testing the functionality
-of positional audio plugins outside of Mumble.
+Mumble Positional Audio Helper is a small tool for testing the functionality of positional audio plugins outside of Mumble.
```

Comment:
Why did you change from limited line length to everything in one line?

While we could debate if our [Coding Guidelines](http://wiki.mumble.info/wiki/Coding_Guidelines) apply here in a markdown readme, we limit lines to 100 characters in our C++ code. Markdown will ignore the newline, and render the result with adequate newlines anyway (at least that is what I expect).